public/playersBetController.js

```diff
@@ -0,0 +1,154 @@
+"use strict";
+export default class PlayersBetController {
+  constructor(board) {
+    this.currentPlayerTurn = 0;
+    this.board = board;
+    this.currentBet = 0;
+
+    this.controlsContainerEle = this.board.boardContainerElem;
+  }
+
+  //Each player must place their bet before the cards are dealt
+
+  //After dealing the cards if the dealer's card is Ace, the players can make an insurance bet, the maximum that can be bet is half the initial bet, it is left on the insurance bet line and it pays 2 to one
+  // if an insurance bet is made, then the player starts his hand normally, that is, with the same options as always
+
+  controlsHtml() {
+    return `
+    <div class='bet-controls'>
+        <div class='bet-controls__pot'>Pot: ${this.activePlayer.pot}</div>
+        <div class='bet-controls__bet'>Current bet: ${this.currentBet}</div>    
+        <button data-amount="25" type='button' class= 'bet-controls__btn btn--25'>
+        25
+        </button>
+        <button data-amount="50" type='button' class= 'bet-controls__btn btn--50'>
+        50
+        </button>
+        <button data-amount="100" type='button' class= 'bet-controls__btn btn--100'>
+        100
+        </button>    
+        <button type='button' disabled class= 'bet-controls__btn btn--clear'>
+        Clear
+        </button>
+        <button type='button' disabled class= 'bet-controls__btn btn--bet'>
+        Confirm bet
+        </button>
+    </div>    
+      `;
+  }
+
+  renderBetControls() {
+    this.controlsContainerEle.insertAdjacentHTML(
+      "beforeend",
+      this.controlsHtml()
+    );
+  }
+
+  initBetController() {
+    this.initPlayers();
+    this.renderBetControls();
+    this.selectDomElements();
+    this.addEventListeners();
+  }
+
+  initPlayers() {
+    this.players = this.board.players;
+    this.updateActivePlayer();
+  }
+
+  selectDomElements() {
+    this.betControls = document.querySelector(".bet-controls");
+    this.bet25 = this.controlsContainerEle.querySelector(".btn--25");
+    this.bet50 = this.controlsContainerEle.querySelector(".btn--50");
+    this.bet100 = this.controlsContainerEle.querySelector(".btn--100");
```

Comment:
Now that you have the `data-amount` attribute for bet controllers you can query them all with `this.controlsContainerEle.querySelectorAll('[data-amount]')` and use this list to iterate over and assign the `eventListeners`